docs/Developers/Newcomers_Guide.rst

```diff
@@ -383,6 +383,26 @@ and clicking ``New Pull Request``.
 
 **Congratulations!** You have now created your first ``Pull Request``!
 
+Now, before having other contributors review your work, the system will do an
+automated code review and will comment the requested improvements (respecting
+coding style and commit message rules, line length and spacing etc.) on your
+Pull Request.
+
+.. important::
+
+    It is **never** allowed to close your Pull Request. **Never!**
+
+    It is **always** possible to modify your Pull Request using
+    `git <https://coala.io/git>`_.
+
+    You must not delete your branch, delete your fork, close your
```

Comment:
You have written vice-versa ```You must not delete your branch, delete your fork``` please research about it and then write because this will go to public when it will be merged.
Write carefully.

Reply:
also you can use stackoverflow for such purposes.Just check [this](https://superuser.com/questions/235860/is-it-safe-to-delete-a-branch-from-a-github-fork-once-its-been-merged-upstream)

Reply:
So, what are you meaning is that it will not cause any harm deleting a branch once it was pulled upstream? Here we are implying that once a Pull Request was open it should not be forcefully closed (either by deleting the branch or deleting the fork) or simply closing the PR without being asked to.

Reply:
Ahh, your comment changed my thought what happens is that if any reader reads this he will understand that deleting a fork is a good practice instead of deleting a branch because it happened with me that i understood it in wrong way so  write ```You should not delete your branch, either your fork, neither close your ``Pull Request`` and open a new one.```

Reply:
Ok, that is understandable. I will try to rephrase a little to get rid of any trace of ambiguity.